addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -108,8 +108,8 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
         }
         // Validate arguments
         // trim text
-        text = text.trim();
-        if ((null == text) || text.isEmpty()) {
+        String trimedText = text.trim();
+        if ((null == trimedText) || trimedText.isEmpty()) {
```

Comment:
This is still wrong trimmed text cannot be null, text however can be null but that will throw an now on trimming